### ISSUE TITLE
Make branch-name configurable

### ISSUE THREAD

**Issue by DanielHabenicht** [restyled-io/restyler#85]
If renovate opens a PR with changes (e.g. `renovate/update-tool-0.x`) and they have not yet been formatted `restyler` opens a new PR (--> `renovate/update-tool-0.x-restyled`). 
So far so good. 

But `renovate` owns the "namespace" `renovate/*` thus treating `renovate/update-tool-0.x-restyled` as if it belongs to `renovate` e.g. autoclosing it. 

What I propose is "namespacing" restyler as well. 

Each new PR from restyler should open under the `resyled/*` "namespace", e.g.: 
`unformatted-pr-example` --> `restyled/unformatted-pr-example`
`renovate/update-tool-0.x` --> `restyled/renovate/update/tool-0.x`

**Comment by pbrisbin** [comment]
I like it. I think we can make it a config key, so the current behavior
might be

```yaml
branch_name: "{base}-restyled"
```

And what you described would be

```yaml
branch_name: "restyled/{base}"
```

WDYT?

**Comment by DanielHabenicht** [comment]
sounds good!

**Comment by pbrisbin** [comment]
Got started on this. It's not too difficult, but there's one sort of logical blocker.

If we need to know the configuration in order to know the Restyled branch name, that means we need to successfully clone the original PR before we can work with any possible Restyled PR. This has a few known ramifications, and who knows how many unknown.

As one example,

1. User opens PR
2. Restyled opens Restyle PR
3. User ignores, merges their PR, branch is auto-deleted
4. Restyler can no longer clone, no longer find the Restyled branch name, no longer find the Restyled PR, and so can no longer close it to clean up after itself

Not great. :thinking:

**Comment by DanielHabenicht** [comment]
Thanks! How do I can configure it? 
I can add your explanation to the wiki afterwards. :)

**Comment by pbrisbin** [comment]
Because of the issues I mentioned, I didn't make it configurable. It just
does restyled/... all the time now.

**Comment by DanielHabenicht** [comment]
ok, cool!

**Comment by DanielHabenicht** [comment]
@pbrisbin this runs into an error now?
https://restyled.io/gh/T-Systems-MMS/repos/phonebook/jobs/245152

**Comment by DanielHabenicht** [comment]
ahh ok, its my fault :D
